src/clusterfuzz/_internal/issue_management/issue_tracker/__init__.py

```diff
@@ -0,0 +1,220 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Issue tracker interface."""
```

Comment:
It's a bit unfortunate that Chromium has chosen the boring-sounding name "issue tracker" and I think it could cause confusion in cases like this - "issue_management/issue_tracker" sounds like generic code across all issue trackers.

Maybe rename this all to `chromium_issue_tracker`? Especially since it's likely to embed details of Chromium specifics, e.g. how we will represent severity and impacted versions.